packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -426,24 +433,97 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         }
     }
 
+    private tooltip(x: number, y: number): void {
+        this.isComputing = true;
+        const startPixel = this.positionXMove;
+        const timeForX = this.getTimeForX(startPixel);
+        let timeLabel: string  | undefined = startPixel.toString();
+        if (this.props.unitController.numberTranslator) {
+            timeLabel = this.props.unitController.numberTranslator(timeForX);
+        }
+        const chartWidth = this.lineChartRef.current.chartInstance.width;
+        const points: any = [];
+        let zeros = 0;
+        this.state.xyData.datasets.forEach((d: any) => {
+            // In case there are less data points than pixels in the chart,
+            // calculate nearest value.
+            const relativeX= (startPixel * chartWidth) / d.data.length;
+            const adjustedX = d.data[relativeX];
+            const rounded = isNaN(adjustedX) ? 0 : (BIMath.round(Number(adjustedX) * 100)/BIMath.round(100));
```

Comment:
The rounding is necessary to round to show the values with 2 decimal cases.
The isNaN check is necessary because without it, sometimes a small tooltip without any text appears a the top left corner of the window. I believe it happens if the mouse moves out of to chart too fast. The isNaN check avoids that.
I prefer to leave this line as it is.